resources/functionMap.php

```diff
@@ -12088,7 +12088,7 @@
 'strtoupper' => ['uppercase-string', 'str'=>'string'],
 'strtr' => ['string', 'str'=>'string', 'from'=>'string', 'to'=>'string'],
 'strtr\'1' => ['string', 'str'=>'string', 'replace_pairs'=>'array'],
-'strval' => ['string', 'var'=>'__stringAndStringable|int|float|bool|resource|null'],
+'strval' => ['string', 'var'=>'mixed'],
```

Comment:
I think it was correct before, see e.g. https://github.com/phpstan/phpstan/issues/9295#issuecomment-1542186125

Reply:
Ok, so the PHP documentation is incorrect, next time I will check the issue tracker first.

I have reverted and commit it again.